src/redux/cards/reducers.ts

```diff
@@ -0,0 +1,89 @@
+
+import { FETCH_CARD, CARD_ERROR, FETCH_CARD_SUCCESS } from './actions/fetch'
+import { CREATE_CARD_SUCCESS, CREATE_CARD_ERROR } from './actions/create'
+import { UPDATE_CARD_SUCCESS, UPDATE_CARD_ERROR } from './actions/update'
+import { REMOVE_CARD_SUCCESS, REMOVE_CARD_ERROR } from './actions/delete'
+
+import { RootAction } from '../RootAction'
+import { ICard } from '../cards/types'
+
+export type State = {
+    card: ICard | null,
+    error: string | null,
+    isProcessing: boolean,
+}
+
+const defaultValue: State = {
+    card: null,
+    error: null,
+    isProcessing: false
+}
+export const reducer = (state: State = defaultValue, action: RootAction) => {
+        switch (action.type) {
+
+            case CARD_ERROR:
+                return {
+                    ...state,
+                    error: action.error,
+                    isProcessing: false
+            }
+            
+            case FETCH_CARD:
+                return {
+                    ...state,
+                    card: null,
+                    error: null,
+                    isProcessing: true
```

Comment:
You change all the state here. This is working, I guess, so do not bother, but weird